docs/tutorials/hostport.md

```diff
@@ -14,13 +14,16 @@ We will go through a small example of deploying a simple Kafka with use of a hea
 A simple deploy could look like this:
 ### Manifest (for clusters without RBAC enabled)
 ```yaml
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: exeternal-dns
 spec:
   strategy:
     type: Recreate
+  selector:
+    matchLabels:
+      app: external-dns
```

Comment:
This `Deployment` is missing the `template.metadata.labels`. Can we just add them?